packages/app/hooks/use-infinite-list-query.ts

```diff
@@ -42,7 +42,7 @@ export const useInfiniteListQuerySWR = <T>(
       // suspense: true,
       refreshInterval,
       revalidateOnMount: true,
-      dedupingInterval: 30000,
+      dedupingInterval: 5000,
```

Comment:
this was required or else doing mutate doesn't perform refetch. I think 5 seconds is good enough for dedup interval. cc - @hirbod let me know what you think!

Reply:
Thats fine. I didn't know it had that many implications

Reply:
yeah, it is not obvios. So if mutate is triggered within 30 secs SWR counts it as a duplicate request. Took a while to debug